### PR TITLE
KEYCLOAK-10677 Fix incorrect tooltip texts

### PR DESCRIPTION
**Pull request by k-tamura**
Please refer to [this ticket](https://issues.jboss.org/browse/KEYCLOAK-10677).

### CODE REVIEW

**Review by abstractj — APPROVED** [review]
@ssilvert looks good to me, but I will wait for your approval.

**Review by ssilvert — APPROVED** [review]
Thanks for the fix!

**Pull request merged**
